explore/dwwc-chain-recursive.ipynb

```diff
@@ -0,0 +1,702 @@
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>metapath</th>\n",
+       "      <th>abbrev</th>\n",
+       "      <th>dwwc</th>\n",
+       "      <th>dwwc-nocache</th>\n",
+       "      <th>recursive</th>\n",
+       "      <th>recursive-nocache</th>\n",
+       "      <th>chain</th>\n",
+       "      <th>chain-nocache</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>(Compound - treats - Disease)</td>\n",
+       "      <td>CtD</td>\n",
+       "      <td>0.024447</td>\n",
+       "      <td>0.009796</td>\n",
+       "      <td>0.014815</td>\n",
+       "      <td>0.010053</td>\n",
+       "      <td>0.005186</td>\n",
+       "      <td>0.009526</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>(Compound - palliates - Disease)</td>\n",
+       "      <td>CpD</td>\n",
+       "      <td>0.005504</td>\n",
+       "      <td>0.004712</td>\n",
+       "      <td>0.005456</td>\n",
+       "      <td>0.004842</td>\n",
+       "      <td>0.004865</td>\n",
+       "      <td>0.004966</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>(Compound - binds - Gene, Gene - upregulates -...</td>\n",
+       "      <td>CbGuD</td>\n",
+       "      <td>0.036345</td>\n",
+       "      <td>0.041072</td>\n",
+       "      <td>0.047547</td>\n",
+       "      <td>0.032535</td>\n",
+       "      <td>0.017528</td>\n",
+       "      <td>0.030719</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>(Compound - binds - Gene, Gene - associates - ...</td>\n",
+       "      <td>CbGaD</td>\n",
+       "      <td>0.022552</td>\n",
+       "      <td>0.019919</td>\n",
+       "      <td>0.021518</td>\n",
+       "      <td>0.020633</td>\n",
+       "      <td>0.020253</td>\n",
+       "      <td>0.019373</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>(Compound - binds - Gene, Gene - downregulates...</td>\n",
+       "      <td>CbGdD</td>\n",
+       "      <td>0.021729</td>\n",
+       "      <td>0.019922</td>\n",
+       "      <td>0.019714</td>\n",
+       "      <td>0.019020</td>\n",
+       "      <td>0.018035</td>\n",
+       "      <td>0.020231</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
```

Comment:
Interesting, when written to TSV, both metapath and abbrev show the abbreviation.